pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining.detailed-output

```diff
@@ -1,15 +1,8 @@
 = Control Plane =
-Assessment:      Completed
-Target Version:  4.16.0-ec.3 (from 4.16.0-ec.2)
-Completion:      100% (36 operators updated, 0 updating, 0 waiting)
-Duration:        3h31m
-Operator Status: 35 Healthy, 1 Available but degraded
+Cluster version is 4.16.0-ec.3
 
 Control Plane Nodes
-NAME                                                  ASSESSMENT   PHASE     VERSION       EST   MESSAGE
-build0-gstfj-m-0.c.openshift-ci-build-farm.internal   Completed    Updated   4.16.0-ec.3   -     
-build0-gstfj-m-1.c.openshift-ci-build-farm.internal   Completed    Updated   4.16.0-ec.3   -     
-build0-gstfj-m-2.c.openshift-ci-build-farm.internal   Completed    Updated   4.16.0-ec.3   -     
+All Nodes are up to date
```

Comment:
Instead of the two lines, I'd emit just one, like

`All control plane nodes successfully updated to $version"